restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/IvrEndpoint.java

```diff
@@ -176,6 +175,28 @@ private void response(final Object message) {
         }
     }
 
+    private void handleAsrr(final Map<String, String> parameters, int code) {
+        if (parameters.containsKey("asrr")) {
+            String asrr = parameters.get("asrr");
+            if (!StringUtils.isEmpty(asrr)) {
+                try {
+                    asrr = new String(Hex.decodeHex(asrr.toCharArray()));
+                } catch (DecoderException e) {
+                    logger.error("asrr parameter cannot be decoded: " + e.getStackTrace());
```

Comment:
to print the stack trace of an exception using Java and Log4J we have to pass exception object as argument.
so statement would be:
`logger.error("asrr parameter cannot be decoded.", e);`